### PR TITLE
fix(managed): fix override label on manual judgement, close artifact …

### PR DESCRIPTION
**Pull request by anotherchrisberry**
…details on escape

These are very minor things I ran into when trying to figure out how the managed delivery code works.

### CODE REVIEW

File: app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx [hunk 1]
```diff
@@ -137,7 +137,7 @@ const constraintsRenderer: { [type in IConstraint['type']]?: IConstraintRenderer
         case 'OVERRIDE_FAIL':
           return (
             <span>
-              <span>Manually {status === 'OVERRIDE_PASS' ? 'approved' : 'rejected'}</span>{' '}
+              <span>Manually {constraint.status === 'OVERRIDE_PASS' ? 'approved' : 'rejected'}</span>{' '}
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx:140]: I guess there's a `window.status` field, which is why Typescript doesn't complain about this?

Comment (ranihorev) [reply app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx:140]: Yeah :(
I opened a PR for that a couple of hours ago

Comment (ranihorev) [reply app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx:140]: https://github.com/spinnaker/deck/pull/9010

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx:140]: ha

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/managed/artifactDetail/constraints/constraintRegistry.tsx:140]: I'll close this and let you deal with the other thing if you want

**Pull request closed without merging**